hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -511,6 +511,8 @@ public function upgradeBundle() {
     $this->upgrade_1029();
     $this->upgrade_1030();
     $this->upgrade_1032();
+    $this->upgrade_1033();
+    $this->upgrade_1034();
```

Comment:
I'm not sure this should be two separate upgrades. The `upgrade_XXX` function adds no meaning when split into two and no clients will ever have `upgrade_1033` applied without `upgrade_1034`. I think it would be better to keep just `upgrade_1033` and split the logic for removing the navigation item and updating the URLs into functions with descriptive names.